components/brave_new_tab_ui/reducers/today/index.ts

```diff
@@ -43,7 +43,7 @@ if (history.state && (history.state.todayArticle || history.state.todayAdPositio
   // TODO(petemill): Type this history.state data and put in an API module
   // see `async/today`.
   defaultState.currentPageIndex = history.state.todayPageIndex as number || 0
-  defaultState.articleScrollTo = history.state.todayArticle as BraveNews.FeedItemMetadata | undefined
+  defaultState.articleScrollTo = history.state.todayArticle
```

Comment:
Can we exclude this please and report in brave-core channel? 🙏

Reply:
Done.